netpyne_ui/netpyne_geppetto.py

```diff
@@ -330,7 +330,7 @@ def simulateNetPyNEModelInGeppetto(self, args):
         use_prev_inst = args.get('usePrevInst', False)
         sim_id = args.get('simId', 0)
         try:
-            self.validate_netParams()
+            # self.validate_netParams()
```

Comment:
A comment to add context about why this was commented and what's the plan to restore it would help understand what's going on